pkg/action/lint.go

```diff
@@ -87,18 +76,18 @@ func lintChart(path string, vals map[string]interface{}, namespace string, stric
 	if strings.HasSuffix(path, ".tgz") {
 		tempDir, err := ioutil.TempDir("", "helm-lint")
 		if err != nil {
-			return linter, err
+			return linter, errors.Errorf("creating temp dir to extract tarball: %s", err.Error())
```

Comment:
All of these should just be `errors.Wrap(err, "message")`

Reply:
Nit: The errors should probably be more clear with a message like "unable to create temp dir..."